build/tools/ExtractRuleDescFromJson/ExtractRuleDescFromJson.csproj

```diff
@@ -18,6 +18,7 @@
   -->
   <ItemGroup Label="SLVS Linked product code">
     <Folder Include="SLVS_LinkedProductCode\" />
+    <Compile Include="..\..\..\src\Core\RegexConstants.cs" Link="SLVS_LinkedProductCode\RegexConstants.cs" />
```

Comment:
The `ExtractRuleDescFromJson.csproj` is one of our build-time tools that extracts data from the jars. As such, it can't have project references to any of our product assemblies, as they haven't been built yet e.g. it can't reference `Core.csproj`.

However, there are a couple of produce source code files we need to reference (e.g. the class used to serialize the rule help json), so the `ExtractRuleDescFromJson` project links directly to the source code files like this.

We now need the regular expression timeout constant, so the project now needs to reference the new constants class too.